tox.ini

```diff
@@ -15,17 +15,17 @@ commands = yapf -dr . {posargs}
 [testenv:isort]
 allowlist_externals = isort
 skip_install = true
-commands = isort -c ./kannon ./test ./integration_test ./example {posargs}
+commands = isort -c ./kannon ./test/unit_test ./test/integration_test ./example {posargs}
```

Comment:
`./test/unit_test ./test/integration_test` can be `./test/` ?

Reply:
Yes, it can be. Fixed!